tsconfig.json

```diff
@@ -11,6 +11,10 @@
         "moduleResolution": "node",
         "noUnusedLocals": true,
         "noUnusedParameters": false,
-        "baseUrl": "src/"
+        "baseUrl": ".",
+        "paths": {
+            "*": ["src/*", "dist/*"]
```

Comment:
Is this to import the json file into the final server.js bundle?

Reply:
Partly, yes. Because the `manifest.json` file exists in a separate directory (`dist`) to the rest of the source code (`src`), we need to [tell the TS compiler](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) where it can look for it.

Reply:
What lead me to question this was the dependency we suddenly have on the dist directory. It's probably not a good idea.

So I'm reconsidering my suggestion of bundling the json in the server.js. I think the initial solution that you had where the file is loaded separately is probably more sane!